ansible-container/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM rhel7
+
+MAINTAINER Aaron Weitekamp <aweiteka@redhat.com>
+
+RUN yum -y install http://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+
+# Not sure if all of these packages are necessary
+# only git and ansible are known requirements
+RUN yum install -y --enablerepo rhel-7-server-extras-rpms net-tools bind-utils tmux git ansible
+
+RUN git clone https://github.com/openshift/openshift-ansible.git /opt/openshift-ansible
+
+ENTRYPOINT ["/usr/bin/ansible-playbook", "/opt/openshift-ansible/playbooks/byo/config.yml"]
```

Comment:
This constrains the user to a specific playbook. We can couple `ENTRYPOINT` with `CMD` for more flexibility.

```
ENTRYPOINT ["/usr/bin/ansible-playbook"]
OPT /opt/openshift-ansible/playbooks/byo/config.yml
```

CMD defines the default playbook. The user can override this by appending another playbook.

```
docker run ansible some/other/playbook.yml
```